packages/augur-ui/src/modules/auth/actions/update-sdk.ts

```diff
@@ -81,10 +85,10 @@ export const createFundedGsnWallet = () => async (
   getState: () => AppState
 ) => {
   try {
+    dispatch(addUpdatePendingTransaction(CREATEAUGURWALLET, TXEventName.Pending));
```

Comment:
this can be taken care of in add-update-transactions if there is a method name to track. this isn't needed here
add to add-update-transactions
```
const ADD_PENDING_QUEUE_METHOD_CALLS = [
  BUYPARTICIPATIONTOKENS,
  REDEEMSTAKE,
  MIGRATE_FROM_LEG_REP_TOKEN,
  BATCHCANCELORDERS,
  TRADINGPROCEEDSCLAIMED,
  MIGRATEOUTBYPAYOUT,
  FORKANDREDEEM
];
```
collection and it will automatically be tracked.